app/controllers/two_factor_authentication/webauthn_verification_controller.rb

```diff
@@ -100,7 +100,7 @@ def credential_ids
     end
 
     def analytics_properties
-      auth_method = if form&.webauthn_configuration&.platform_authenticator || params[:platform]
+      auth_method = if form&.webauthn_configuration&.platform_authenticator || params[:platform] == 'true'
```

Comment:
let's do a string cast? in case we have any specs that pass in a literal `true` value (or ever use JSON params that would have a literal `true`)
```suggestion
      auth_method = if form&.webauthn_configuration&.platform_authenticator || params[:platform].to_s == 'true'
```